match/lib/match/utils.rb

```diff
@@ -1,10 +1,21 @@
 module Match
   class Utils
     def self.import(item_path, keychain)
-      command = "security import #{item_path.shellescape} -k ~/Library/Keychains/#{keychain.shellescape}"
+      # Existing code expects that a keychain name will be expanded into a default path to Libary/Keychains
+      # in the user's home directory. However, this will not allow the user to pass an absolute path
+      # for the keychain value
+      keychain_path = File.join(Dir.home, 'Library', 'Keychains', keychain)
+
+      # So, if the passed value can't be resolved as a file in Library/Keychains, just use it as-is
+      # as the keychain path.
+      keychain_path = keychain unless File.exist?(keychain_path)
```

Comment:
Shouldn't we check if the `keychain` exists too? Or is that not possible, as Apple does some path magic when accessing the Keychain? Can we do something like listing all available keychains?